src/com/magento/idea/magento2uct/ui/ConfigurationDialog.java

```diff
@@ -27,13 +27,7 @@
 import java.awt.event.WindowAdapter;
 import java.awt.event.WindowEvent;
 import java.util.Objects;
-import javax.swing.JButton;
-import javax.swing.JCheckBox;
-import javax.swing.JComboBox;
-import javax.swing.JComponent;
-import javax.swing.JLabel;
-import javax.swing.JPanel;
-import javax.swing.KeyStroke;
+import javax.swing.*;
```

Comment:
Please, do not use wildcard imports.

Reply:
Thanks for your code review. I fixed. Please check.